CONTRIBUTING.md

```diff
@@ -112,11 +112,11 @@ If your IDE supports the [Language Server Protocol (LSP) specification](https://
 
 ## Starting the demos
 
-First, clone the repo and install the NPM dependencies from within the `calcite-components` directory:
+First, clone the repo, install the NPM dependencies:
```

Comment:
Nitpick: I feel like we need an `"and then"` between cloning and installing deps. I'll leave this up to you.